<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SSMenu</class>
 <widget class="QDialog" name="SSMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>50</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget_controller" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#widget_controller
{
	background-color: rgba(60, 65, 68,200);
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_open">
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{ 
image: url(:image/open_normal.png);
border-radius:0px; 
} 
QPushButton:hover{ 
image: url(:image/open_focus.png);
border-radius:0px; 
} 
QPushButton:pressed{ 
image: url(:image/open_normal.png);
border-radius:0px; 
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_close">
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{ 
image: url(:image/closeAll_normal.png);
border-radius:0px; 
} 
QPushButton:hover{ 
image: url(:image/closeAll_focus.png);
border-radius:0px; 
} 
QPushButton:pressed{ 
image: url(:image/closeAll_normal.png);
border-radius:0px; 
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>6</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_play">
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{ 
image: url(:image/start_normal.png);
border-radius:0px; 
}  
QPushButton:hover{ 
image: url(:image/start_focus.png);
border-radius:0px; 
} 
QPushButton:pressed{ 
image: url(:image/start_normal.png);
border-radius:0px; 
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_pause">
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{ 
image: url(:image/pause_normal.png);
border-radius:0px; 
} 
QPushButton:hover{ 
image: url(:image/pause_focus.png);
border-radius:0px; 
} 
QPushButton:pressed{ 
image: url(:image/pause_normal.png);
border-radius:0px; 
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>6</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_stop">
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{ 
image: url(:image/stop_normal.png);
border-radius:0px; 
}  
QPushButton:hover{ 
image: url(:image/stop_focus.png);
border-radius:0px; 
} 
QPushButton:pressed{ 
image: url(:image/stop_normal.png);
border-radius:0px; 
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>6</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="VideoSlider" name="horizontalSlider_2">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QSlider::groove:horizontal {
border: 0px solid #bbb;
}

QSlider::sub-page:horizontal {
background-color: #ff9900;
border-bottom-left-radius: 2px; 
border-top-left-radius: 2px; 

border-radius: 2px; 
margin-top:5px;
margin-bottom:5px;
}

QSlider::add-page:horizontal {
background-color: rgba(228, 228, 228, 50);v
border: 0px solid #777;
border-radius: 2px;
margin-top:5px;
margin-bottom:5px;
}
/*
QSlider::handle:horizontal {
background: rgb(255,153,102); 
border: 1px solid rgb(255,153,102); 
width: 14px;
height:10px;
border-radius: 7px;
margin-top:2px;
margin-bottom:2px;
}

QSlider::handle:horizontal:hover {
background: rgb(255,128,6); 
border: 1px solid rgba(102,102,102,102);
border-radius: 7px;
}*/

QSlider::sub-page:horizontal:disabled {
background: #bbb;
border-color: #999;
}

QSlider::add-page:horizontal:disabled {
background: #eee;
border-color: #999;
}

QSlider::handle:horizontal:disabled {
background: #eee;
border: 1px solid #aaa;
border-radius: 2px;
}
</string>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>3</number>
          </property>
          <item>
           <widget class="QLabel" name="label_currenttime">
            <property name="styleSheet">
             <string notr="true">QLabel{
	border-radius:0px;
	color: #F0F0F0;
	background-color:rgba(0,0,0,0);
	border-style:none;
font-family:&quot;微软雅黑&quot;;
font-size:16px;
}
</string>
            </property>
            <property name="text">
             <string>00:00:00</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="styleSheet">
             <string notr="true">QLabel{
	border-radius:0px;
	color: #F0F0F0;
	background-color:rgba(0,0,0,0);
	border-style:none;
font-family:&quot;微软雅黑&quot;;
font-size:16px;
}
</string>
            </property>
            <property name="text">
             <string>/</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_totaltime">
            <property name="styleSheet">
             <string notr="true">QLabel{
	border-radius:0px;
	color: #F0F0F0;
	background-color:rgba(0,0,0,0);
	border-style:none;
font-family:&quot;微软雅黑&quot;;
font-size:16px;
}
</string>
            </property>
            <property name="text">
             <string>00:00:00</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_volume">
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{ 
image: url(:/image/volume_normal.png);
border-radius:0px; 
padding:5px;
}  
QPushButton:hover{ 
image: url(:image/stop_focus.png);
border-radius:0px; 
padding:3px;
} 
QPushButton:!checked{ 
image: url(:/image/volume_normal.png);
} 
QPushButton:checked{ 
image: url(:/image/volume_mute.png);
} 

</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider_volume">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QSlider::groove:horizontal {
border: 0px solid #bbb;
}

QSlider::sub-page:horizontal {
background: rgb(235,97,0);
border-radius: 0px;
margin-top:8px;
margin-bottom:8px;
}

QSlider::add-page:horizontal {
background: rgb(255,255, 255);
border: 0px solid #777;
border-radius: 2px;
margin-top:8px;
margin-bottom:8px;
}

QSlider::handle:horizontal {
background: rgb(255,153,102); 
border: 1px solid rgb(255,153,102); 
width: 8px;
height:8px;
border-radius: 2px;
margin-top:6px;
margin-bottom:6px;
}

QSlider::handle:horizontal:hover {
background: rgb(255,128,6); 
border: 1px solid rgba(102,102,102,102);
border-radius: 7px;
}

QSlider::sub-page:horizontal:disabled {
background: #bbb;
border-color: #999;
}

QSlider::add-page:horizontal:disabled {
background: #eee;
border-color: #999;
}

QSlider::handle:horizontal:disabled {
background: #eee;
border: 1px solid #aaa;
border-radius: 4px;
}

</string>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="singleStep">
           <number>0</number>
          </property>
          <property name="pageStep">
           <number>0</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_volume">
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
	border-radius:0px;
	color: #F0F0F0;
	background-color:rgba(0,0,0,0);
	border-style:none;
font: 16px &quot;微软雅黑&quot;;

}
</string>
          </property>
          <property name="text">
           <string>100</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>6</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_model">
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{ 
image: url(:image/model_normal.png);
border-radius:0px; 
} 
QPushButton:hover{ 
image: url(:image/model_focus.png);
border-radius:0px; 
} 
QPushButton:pressed{ 
image: url(:image/model1_normal.png);
border-radius:0px; 
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_fullscreen">
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{ 
image: url(:image/fullscreen_normal.png);
border-radius:0px; 
} 
QPushButton:hover{ 
image: url(:image/fullscreen_focus.png);
border-radius:0px; 
} 
QPushButton:pressed{ 
image: url(:image/fullscreen1_normal.png);
border-radius:0px; 
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_selectchannel">
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{ 
image: url(:image/param_normal.png);
border-radius:0px; 
} 
QPushButton:hover{ 
image: url(:image/param_focus.png);
border-radius:0px; 
} 
QPushButton:pressed{ 
image: url(:image/param1_normal.png);
border-radius:0px; 
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_edit">
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{ 
image: url(:image/edit_normal.png);
border-radius:0px; 
} 
QPushButton:hover{ 
image: url(:image/edit_focus.png);
border-radius:0px; 
} 
QPushButton:pressed{ 
image: url(:image/edit1_normal.png);
border-radius:0px; 
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VideoSlider</class>
   <extends>QSlider</extends>
   <header>Widget/VideoSlider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
